index.html

```diff
@@ -0,0 +1,187 @@
+
+    <div class="enclose"></div>
+
+    <div class="program">
+      <div class="section-text">
+        <h4>What we do</h4>
+        <div class="program-line"></div>
+      </div>
+
+      <div class="card-container">
+        <div class="card-content">
+          <span class="material-symbols-outlined card-margin">
+            computer
+          </span>
+            <p class="card-margin technology">Repairs</p>
+          <p class="content-paragraph">Computer repairs and maintenance for better system performance.
+          </p>
+        </div>
+  
+        <div class="card-content">
+          <span class="material-symbols-outlined card-margin">
+            tools_installation_kit
+          </span>
+            <p class="card-margin technology">Install</p>
+          <p class="content-paragraph">Computer repairs and maintenance for better system performance.
+          </p>
+        </div>
+  
+        <div class="card-content">
+          <span class="material-symbols-outlined card-margin">
+            quiz
+            </span>
+            <p class="card-margin technology">Consult</p>
+          <p class="content-paragraph">Computer repairs and maintenance for better system performance.
+          </p>
+        </div>
+  
+        <div class="card-content">
+          <span class="material-symbols-outlined card-margin">
+            point_of_sale
+            </span>
+            <p class="card-margin technology">Sales</p>
+          <p class="content-paragraph">Computer repairs and maintenance for better system performance.
+          </p>
+        </div>
+  
+        <div class="card-content">
+          <span class="material-symbols-outlined card-margin">
+            cast_for_education
+            </span>
+            <p class="card-margin technology">Training</p>
+          <p class="content-paragraph">Computer repairs and maintenance for better system performance.
+          </p>
+        </div>
+  
+      </div>
+      <a href="" class="more-link">See The Whole Services We Offer</a>
+
+
+
+      <button class="btn">Sign up for business</button>
+
+    </div>
+
+    <div class="client-section">
+      <h4>What our clients say...</h4>
+      <div class="program-line"></div>
+
+      <!-- dynamic content -->
+
+
+      <div class="main-content"></div>
+    </div>
+
+    <section>
+      <div class="partners">
+        <h5 class="partner-header">Training Partners</h5>
+        <div class="program-line"></div>
+        <div class="partner-div">
+           <div class="partner-image">
+            <img src="assets/partner/amazonfinal.png" alt="" width="110px">
+            <img src="assets/partner/cisco.png" alt="" width="110px" height="60px">
+            <img src="assets/partner/facebook.png" alt="" width="110px" height="50px">
+            <img src="assets/partner/google.png" alt="" width="110px" height="60p">
+           </div>
+        </div>
+      </div>
+    </section>
+
+    <div class="show">
+      <p>See More</p>
```

Comment:
- [OPTIONAL] Please make sure, when the user clicks on this button it displays more items as per the project requirements. :recycle: 
__Please note that this point is optional only because the previous reviewer didn't mention it, but it is crucial to make your project complete__

- Your design:
![yourP](https://user-images.githubusercontent.com/97834160/210793995-da536df7-de7b-4aeb-97af-e068cafd3767.png)

- Behance design:
![Bp](https://user-images.githubusercontent.com/97834160/210793946-78712d62-3127-4224-80b3-8373ddd04b77.png)